task/prefetch-dependencies-oci-ta/0.1/prefetch-dependencies-oci-ta.yaml

```diff
@@ -143,15 +157,159 @@ spec:
           # https://github.com/containerbuildsystem/cachi2/issues/577
           yq 'del(.goproxy_url)' <<<"${CONFIG_FILE_CONTENT}" >/mnt/config/config.yaml
         fi
-    - name: prefetch-dependencies
+    - name: check-prefetch-input
       image: quay.io/redhat-appstudio/cachi2:0.13.0@sha256:eb34cfe3fea20997eebd8164dc93eedb2fd7a60dc1fb4afcc1b1ff43df9d6667
+      env:
+        - name: INPUT
+          value: $(params.input)
+      script: |
+        if [ -z "${INPUT}" ]; then
+          # Confirm input was provided though it's likely the whole task would be skipped if it wasn't
+          echo "No prefetch will be performed because no input was provided for cachi2 fetch-deps"
+          echo "skip" >/shared/skip
+        fi
```

Comment:
this is just preserving existing behavior.  https://github.com/konflux-ci/build-definitions/blob/6f4a50deb863050574a5422e89279aa43b1399ae/task/prefetch-dependencies/0.1/prefetch-dependencies.yaml#L101

Reply:
...GitHub is either acting up or I can't mark the thread as resolved. Maybe because I'm not part of the org? @brianwcook if you can do that, then you can resolve this thread.